projects/reservoir-protocol/index.js

```diff
@@ -0,0 +1,31 @@
+const ADDRESSES = require('../helper/coreAssets.json');
+
+const rUSD = '0x09D4214C03D01F49544C0448DBE3A27f768F2b34';
+const srUSD = '0x738d1115B90efa71AE468F1287fc864775e23a31';
+const termIssuer = '0x128D86A9e854a709Df06b884f81EeE7240F6cCf7';
+
+const tvl = async (api) => {
+
+    const trUSDTotalDebt = await api.call({ target: termIssuer, abi: 'function totalDebt() external view returns (uint256)' });
+    api.add(termIssuer, trUSDTotalDebt);
+
+    const rUSDTotalSupply = await api.call({ target: rUSD, abi: 'function totalSupply() external view returns (uint256)' });
+    api.add(rUSD, rUSDTotalSupply);
+
+    const srUSDTotalSupply = await api.call({ target: srUSD, abi: 'function totalSupply() external view returns (uint256)' });
+    api.add(srUSD, srUSDTotalSupply / 1e12);
+}
+
+module.exports = {
+    ethereum: {
+        tvl,
+        borrowed: async (api) => {
+            const borrows = await api.call({
```

Comment:
amount not lent out would be tvl, rest is borrowed. so, need to swap the keys here

Reply:
@g1nt0ki Check now if you don't mind, changed our TVL to our full capital held by the protocol. Let me know if any issues please :)